spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/actions/TestCreateActions.java

```diff
@@ -516,17 +527,16 @@ public void testProperties() throws Exception {
     expectedProps.put("dogs", "sundance");
 
     for (Map.Entry<String, String> entry : expectedProps.entrySet()) {
-      Assert.assertTrue(
-          "Created table missing property " + entry.getKey(),
-          table.properties().containsKey(entry.getKey()));
-      Assert.assertEquals(
-          "Property value is not the expected value",
-          entry.getValue(),
-          table.properties().get(entry.getKey()));
+      assertThat(table.properties())
```

Comment:
these two assertions can be simplified to a single one using `containsEntry(...)`